### PR TITLE
Fix issue #268 : wrong shirt colors in the character selection

### PR DESCRIPTION
**Pull request by Racater**
This pull request fixes the issue #268 .
Now the characters will get the correct shirt color for shirts such as the space suit from the server following character creation (dark green and medium blue).
The error was probably due to a wrong copy and paste in the code.
Has been tested locally.

### CODE REVIEW

**Comment by averysumner** [comment]
Looks good to me.

**Comment by Jettford** [comment]
Maybe this should be replaced with a CDClient query such as this implementation of it. https://github.com/UchuServer/Uchu/blob/92894ace6f0f83ff1c9725b2bde58dfa4e883481/Uchu.Char/Handlers/CharacterHandler.cs#L97-L110

**Review by DarwinAnim8or — APPROVED** [review]

**Comment by DarwinAnim8or** [comment]
@Racater hadn't seen your latest commit before reviewing, have you tested this works?

**Comment by Racater** [comment]
@Jettford following your comment, I rewrote and simplified the code for the shirt at character creation. This way we remove this list of hardcoded values.

**Comment by Racater** [comment]
This has been tested to work on my local server with all edge cases I could think of (first shirt style, last shirt style, different colors, etc.)

File: dGame/UserManager.cpp [hunk 1]
```diff
@@ -566,114 +566,34 @@ void UserManager::LoginCharacter(const SystemAddress& sysAddr, uint32_t playerID
...
-		break;
-	}
-	}
-
-	// Initialize another variable for the shirt color
-	uint32_t editedShirtColor = shirtID;
-
-	// This will be the final shirt ID
-	uint32_t shirtIDFinal;
+uint32_t getShirtColorId(uint32_t color) {
```
Comment (MickVermeulen) [inline dGame/UserManager.cpp:569]: This should be CamelCase

File: dGame/UserManager.cpp [hunk 2]
```diff
@@ -566,114 +566,34 @@ void UserManager::LoginCharacter(const SystemAddress& sysAddr, uint32_t playerID
...
+	
+	// get the index of the color in shirtColorVector
+	auto colorId = std::find (shirtColorVector.begin(), shirtColorVector.end(), color);
```
Comment (MickVermeulen) [inline dGame/UserManager.cpp:572]: `std::find (` should be `std::find(`

**Review by MickVermeulen — CHANGES_REQUESTED** [review]
Have some code style comments.

File: dGame/UserManager.cpp (hunk 1 above)
Comment (Racater) [reply dGame/UserManager.cpp]: done

File: dGame/UserManager.cpp (hunk 2 above)
Comment (Racater) [reply dGame/UserManager.cpp]: fixed

**Review by MickVermeulen — APPROVED** [review]
Yeah LGTM, @DarwinAnim8or you assigned yourself for review, do you still plan on doing that or was it a misclick (given that you already approved before :P)

**Review by DarwinAnim8or — APPROVED** [review]

**Pull request merged**